tests/AgDatabaseMove.Unit/FileToolsTest.cs

```diff
@@ -7,38 +7,35 @@ namespace AgDatabaseMove.Unit
 
   public class BackupFileToolsTest
   {
-    public static IEnumerable<object[]> UrlFileExamples => new List<object[]> {
-      new object[] { "https://hello/a.bak" },
-      new object[] { "https://hello/a.full" },
-      new object[] { "https://storage-account.blob.core.windows.net/container/file.trn" },
-      new object[] { "https://hello/a.diff" },
-      new object[] { "https://a.diff" },
-      new object[] { "https://1/2/3/4/5/a.diff" },
-      new object[] { "https://storage-account.blob.core.windows.net/container/file.bad" },
-      new object[]
-        { "https://storage-account.blob.core.windows.net/container/sql/db_name/backup_2020_09_02_170003_697.trn" },
-      new object[] { "http://hello/a.bak" }
-    };
-
-    public static IEnumerable<object[]> NonUrlFileExamples => new List<object[]> {
-      new object[] { @"c:\hello\a.bak" },
-      new object[] { @"\\abc\hello/a.bak" },
-      new object[] { "https://storage-account.blob.core.windows.net/container" },
-      new object[] { "http://storage-account.blob.core.windows.net/container" }

```

Comment:
you aren't using these cases anymore. I'm pretty sure the last two here would break your tests :p
Add them back please

Reply:
Will add back the test case for `c:\hello\a.bak` test case - that still works

For `\\abc\hello/a.bak` - unc urls are a valid URL here
_Quick note:_ URI class is a bit more intelligent and flips the slashes
This [article](https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/unc-path-recognition-unix) explains it on .Net 5, but looks like they have something similar for .Net core 2.0 too - [docs](https://docs.microsoft.com/en-us/dotnet/api/system.uri.localpath?view=netcore-2.0#remarks)

For `"https://storage-account.blob.core.windows.net/container"` and `"http://storage-account.blob.core.windows.net/container"` - pls see comment above - if needed, I'll add them back

Reply:
This comment only meant to discuss line 26 & 27, I think the first two still existed.
Correct, the last two are not valid files

Reply:
moved them back as invalid test data - tests pass